lib/Wikia/src/CircuitBreaker/CircuitBreakerStorage.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Wikia\CircuitBreaker;
+
+use Wikia\Service\Helios\ClientException;
```

Comment:
This class is also used for Rabbit, right? If so, I think we shouldn't be basing it off the Helios client exception, but something more generic (or something specific to the CB namespace).

Reply:
Fix in 637cc452d01